ocw/dataset_processor.py

```diff
@@ -173,6 +176,11 @@ def subset(subregion, target_dataset):
     # Get subregion indices into subregion data
     dataset_slices = _get_subregion_slice_indices(subregion, target_dataset)
 
+    if subregion_name:
```

Comment:
Hey Kim,

You can simplify this if-else block down to simply

```
if not subregion_name:
    subregion_name = target_dataset.name
```

You'll need to update the line below where you pass the name as well if you change this of course =D